addons/l10n_uy_website_sale/data/ir_model_fields.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <function model="ir.model.fields" name="formbuilder_whitelist">
+        <value>res.partner</value>
+        <value eval="['l10n_latam_identification_type_id']"/>
```

Comment:
Not sure about that part. Is it not erasing all existing values?

Reply:
@kcv-odoo Up

Reply:
No, This method just make this field whitelist when this module is installed without changing any other fields on `res.partner` :see_no_evil: 

(It only needed up-to 18.1 as we added new structure to do this in portal module)